comms/src/message/message_data.rs

```diff
@@ -49,6 +52,7 @@ impl MessageData {
     pub fn try_into_frame_set(self) -> Result<FrameSet, MessageError> {
         let mut frame_set = Vec::new();
         frame_set.push(self.source_node_id.as_ref().to_vec());
+        frame_set.extend(self.forwardable.to_binary());
```

Comment:
`to_binary` returns a Result - bit confused why this compiles ;)

Reply:
Not sure either, all the tests pass and it compiles